package.json

```diff
@@ -22,5 +15,9 @@
   "bugs": {
     "url": "https://github.com/YaroslavOvdii/fliplet-widget-slider/issues"
   },
-  "homepage": "https://github.com/YaroslavOvdii/fliplet-widget-slider#readme"
+  "homepage": "https://github.com/YaroslavOvdii/fliplet-widget-slider#readme",
```

Comment:
`homepage` and `bugs` don't seem to link to relevant places, can you remove them?

Reply:
Yes, I'll remove them.

Reply:
@tonytlwu 
Updated